FullAdder.ipynb

```diff
@@ -0,0 +1,300 @@
+{
```

Comment:
<blockquote>This process has been shown in class both classically and on a quantum computer. </blockquote><p>This would be a good location for a reference. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ubsuny/QC_fullAdder/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>